Dockerfile

```diff
@@ -1,9 +1,7 @@
 FROM node:14-alpine
-RUN mkdir -p /home/node/app/node_modules && chown -R node:node /home/node/app
 WORKDIR /home/node/app
-COPY package*.json ./
-USER node
-RUN npm install
 COPY --chown=node:node . .
+RUN npm install
+RUN npm run build
 EXPOSE 3000
-CMD [ "/home/node/app/run.sh" ]
+CMD npm run start
```

Comment:
Getting rid of the run.sh file is good since it wasn't really needed, but it now runs start instead of production like it did before. This will change the behavior of the server when it is running on Logica.